### DISCUSSION TITLE
Aviation -cockpit settings for GoPro Hero13

### DISCUSSION THREAD

**Discussion by User-10-0** [gopro/labs#1126] [Feature Requests]
Is there a way to have custom settings on Protune to avoid overheat and have a better battery life also to use ND8 filter? 
In the summer time there is no cold air flow so the result is overheated camera.
Any pilots here with similar experiences any help?

Thanks in advance.

**Comment by dnewman-gpsw** [comment]
Heat will be resolution, framerate, bit-rate and bit-depth dependent.  HS on/off us about 0.1W, mostly insignificant.  Protune settings will not alter power usage.  Power used is the mount of heat that needs to be dissipated. Less power is longer run times, in static air (you can always add a mini fan to solve most heat issues.)

All there are with the LCD at full brightness.  Screen off is ~0.3W lower.

1080p30 standard bitrate 8-bit HS off uses ~4.0W.
1080p60 standard bitrate 8-bit HS off uses ~5.0W.
4K24 standard bitrate 8-bit HS off uses ~4.8W.
4K24 standard bitrate 8-bit HS on uses ~4.9W.
4K30 standard bitrate 8-bit uses ~5.3W.
4K60 standard bitrate 8-bit uses ~6.2W.
4K60 standard bitrate 10-bit uses ~6.4W.
4K60 high bitrate 10-bit uses ~6.6W.
4K60 BITR=200 10-bit uses ~6.7W.
5.3K30 8:7 BITR=200 10-bit uses ~7.0W.
5.3K60 BITR=200 10-bit uses ~7.7W.

**Reply by User-10-0** [reply]
Thank you for sharing. What i ment is also the quality of the video acording with those settings. 
The mini fan is not an option because the camera is sticked on the canopy.

**Reply by dnewman-gpsw** [reply]
Quality?  If you are shooting in the day, quality will automatically use ISO 100, so great.

**Reply by User-10-0** [reply]
What do you think?
Filming during day light, with light differences from cockpit and outside the best setting to avoid the Hero 13 camera overheating in that environment counting that there is no airflow and the temperature could be high (86-95F degrees).
ND8 filter for propeller with settings 16:9|4K | 25 frames frames | HLG HDR | Wide | Hypersmooth AutoBoost | Motion Blur Auto | Bit Depth 10-Bit | Bit Rate High | Shutter Auto Cinematic | EV Comp 0 | White Balance Auto | Sharpness High | Denoise High | ColorVibrant 

Thank you in advance

**Reply by dnewman-gpsw** [reply]
Just don't use HLG HDR, that will use more power. With standard 4K 25 you could when use 8-bit for longer run time in a hot environment. Use Sharpness Medium, and EV -0.5 for better quality settings.
